docs/UserGuide.md

```diff
@@ -612,11 +586,11 @@ Example: `c/find n/Alex`
 
 ## FAQ
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
-the data of your previous Socius home folder.
+**A**: Install Socius in the other computer. Export your Socius contacts list to a JSON file and copy this JSON file to 
+the same folder in the other computer. Import your contacts list from the JSON file to Socius in the other computer.
```

Comment:
What `same folder` means in `same folder in the other computer` ?

Reply:
Fixed. Replaced it with a simple 5 step guide instead. I prefer not to use the original sentence since it is quite vague.